lib/Elastica/Query/Ids.php

```diff
@@ -77,7 +77,7 @@ public function setType($type)
             return $this;
         }
 
-        $this->_params['type'] = $type;
+        $this->_params['type'] = (array)$type;
```

Comment:
Before my changes:
```php
$ids = new Ids();
$ids->setType('some');
$ids->addType('other'); // Fatal error; // because we try [] to string variable
```